xbmc/platform/win32/storage/discs/Win32DiscDriveHandler.cpp

```diff
@@ -0,0 +1,98 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "Win32DiscDriveHandler.h"
+
+#include "utils/log.h"
+
```

Comment:
remove blank line / include order

Reply:
this comes from clang-format and is automatically re-written if you change it. I guess it has to do with the number of subfolders. They are only ordered alphabetically afterwards (0 subfolders, 1 subfolder, 2 subfolders).